modules/nw-enabling-hsts-global.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id="nw-enabling-hsts-global_{context}"]
+= Enabling HTTP strict transport security globally
```

Comment:
```suggestion
= Enforcing HTTP strict transport security globally
```
It's more accurate to say "enforcing" because it does not actually change anything automatically.  It just adds a policy that will let the user know if they have tried to configure a route that doesn't follow the HSTS policy.